pkgs/top-level/all-packages.nix

```diff
@@ -20701,6 +20701,11 @@ with pkgs;
 
   iterm2 = callPackage ../applications/misc/iterm2 {};
 
+  sequeler = callPackage ../applications/misc/sequeler {
+    libgee = libgee_0_8;
```

Comment:
We should get rid of this, `gnome3.libgee` is the current version.

Reply:
Fixed.